test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/BasicWebSite.Area1.RemoteAttribute_Home.Create.html

```diff
@@ -83,10 +91,10 @@
 
     </div>
 
-    <script src="/lib/jquery/jquery.js"></script>
+    <script src="/javascript/jquery-2.1.4.min.js"></script>
```

Comment:
:ok: but where is the latest Unobtrusive (3.2.6) on the CDN? I can only find the older 5.2.3  (yeah, the "new" version is odd) and earlier files listed [on www.asp.net](https://www.asp.net/ajax/cdn#ASPNET_MVC_Releases_on_the_CDN_16).